subgraphs/atlas-mine/schema.graphql

```diff
@@ -93,6 +93,23 @@ type Reward @entity {
   user: User!
 }
 
+"NFTs staked as boosts"
+type StakedToken @entity {
+  id: Bytes!
+
+  "Staking user"
+  user: User!
+
+  "NFT contract address"
+  contract: Bytes!
+
+  "NFT token ID"
+  tokenId: BigInt!
```

Comment:
Opposed to making this `Int!` instead of `BigInt!`?

Reply:
Not opposed